src/components/GroupedBarChart.js

```diff
@@ -12,6 +13,11 @@ import {
 const getMax = data => data[0].reduce((p, { y }) => (p > y ? p : y), 0)
 
 export default ({ data }) => {
+  const { theme, colorMode } = useThemeUI()
+  const colors =
+    colorMode !== theme.initialColorMode
+      ? theme.colors.modes[colorMode]
+      : theme.colors
```

Comment:
@jxnblk A native way to get the current colorset out of Theme UI would be amazing (unless I'm missing an existing method?)

Otherwise, could move this to a helper

Reply:
Just now realized that this is a bug with Theme UI. The `theme.colors` object *should* be available in context here, and I've opened a PR with a fix https://github.com/system-ui/theme-ui/pull/226

Reply:
To clarify, `theme.colors` is available, but it was just the raw object, not the current set dependent on theme.

This was my first time using Theme UI—is there a better way to get access to theme colors than the hook?

Reply:
That's the preferred way, you just had to do the work around on color modes due to a bug in  Theme UI. The latest version should now have the correct color set based on the active color mode so we don't need to do that dance anymore.